### PR TITLE
Fe listing edit listing

### PR DESCRIPTION
**Pull request by Gonlonge**
Edit listing page OK

### CODE REVIEW

File: pages/listings/listing/editListing.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Noroff Edit Job Listing</title>
+  </head>
+  <body class="d-flex flex-column flex-fill min-vh-100 gap-4" id="homepage">
+    <header></header>
+    <main class="container-fluid d-flex flex-fill flex-lg-row justify-content-center">
+      <div class="mt-5 mb-5" id="newListing" data-bs-backdrop="static" tabindex="-1">
+        <div class="border">
+          <!-- Waiting on API docs for validation requirements -->
+          <form class="m-2" id="createNewListing">
```
Comment (Jenfer93) [inline pages/listings/listing/editListing.html:16]: Maybe change the ID to something like editLisiting, so the same ID isn't used again :)

**Review by Jenfer93 — COMMENTED** [review]
Other than the one comment I think the form looks very good! Good job :) Wait for Åke to review it before any merge is done :)

File: pages/listings/listing/editListing.html (hunk 1 above)
Comment (Gonlonge) [reply pages/listings/listing/editListing.html:16]: I'll fix it 👍

**Review by akeek — CHANGES_REQUESTED** [review]
All the tests I have written passes. But the edit and delete functions are identical. I think there should be som more input to the editSingleListing-function.
Also, there should be a method to each function? But the API-documentation have not provided that?

**Pull request closed without merging**